app/client/sagas/authSagas.js

```diff
@@ -67,6 +67,7 @@ function* watchSaveOrgId() {
   yield takeEvery(UPDATE_ACTIVE_ORG, saveOrgId);
 }
 export function* logout() {
+    yield call(removeTFAToken);
```

Comment:
tfa token wasn't removed on logout

Reply:
Not removing TFA token on logout was by design - if the user has selected to remember the computer, it makes sense that they'd want this even after logging out (this is how AWS/Gmail etc behave)

Reply:
Hm, that makes sense. 

It was causing issues when I then tried to login with another user account though?

Reply:
mmmm yeah good point. Maybe we should save tfa tokens as a dictionary indexed by user id's....

Reply:
so should this be updated so that a TFA token isn't removed (and just leave the edgecase bug in for the time being)?